data/add_coordinates.py

```
@@ -0,0 +1,210 @@
+    """
+    Отримує координати для локації.
+
+    Args:
+        location_name: Назва локації
+
+    Returns:
+        (latitude, longitude)
+    """
+
+    # Якщо не знайдено у статичному мапінгу, використовуємо API
+    coords = get_coordinates_from_nominatim(location_name)
```

Ruff (0.12.2)

Undefined name `location_name`

(F821)